pkg/controller/direct/cloudbuild/workerpool_controller.go

```diff
@@ -302,7 +302,23 @@ func (a *Adapter) Update(ctx context.Context, u *unstructured.Unstructured) erro
 }
 
 func (a *Adapter) Export(ctx context.Context) (*unstructured.Unstructured, error) {
```

Comment:
Idea: (unrelated to this PR) we could have this return a runtime.Object, and then have the caller convert it to unstructured if needed.  Would save a few lines in each Export method.

Reply:
Yes! This also makes the golden export more "correct". Right now, I have to re-write the name and namespace for golden export but this could miss other metadata like labels and annotations.